src/tools/illink/src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -54,63 +55,11 @@ public override void Initialize (AnalysisContext context)
 					CheckMatchingAttributesInInterfaces (symbolAnalysisContext, typeSymbol);
 				}, SymbolKind.NamedType);
 
-				context.RegisterOperationAction (operationContext => {
-					var methodInvocation = (IInvocationOperation) operationContext.Operation;
-					CheckCalledMember (operationContext, methodInvocation.TargetMethod, incompatibleMembers);
-				}, OperationKind.Invocation);
-
-				context.RegisterOperationAction (operationContext => {
-					var objectCreation = (IObjectCreationOperation) operationContext.Operation;
-					var ctor = objectCreation.Constructor;
-					if (ctor is not null) {
-						CheckCalledMember (operationContext, ctor, incompatibleMembers);
-					}
-				}, OperationKind.ObjectCreation);
-
 				context.RegisterOperationAction (operationContext => {
 					var fieldReference = (IFieldReferenceOperation) operationContext.Operation;
 					CheckCalledMember (operationContext, fieldReference.Field, incompatibleMembers);
```

Comment:
This seems to be the last peice which needs CheckCalledMember (and all the infra that brings in). Would it make sense to move this to the DAM analyzer as well? I wonder how this plays with the rest of it. Don't we see field references in code in assignments and such anyway?